src/bin/julialauncher.rs

```diff
@@ -300,6 +302,181 @@ fn get_override_channel(
     }
 }
 
+fn get_project(args: &Vec<String>) -> Option<PathBuf> {
+    let mut project_arg: Option<String> = None;
+    for (_, arg) in args.iter().enumerate() {
+        if arg.starts_with("--project=") {
+            project_arg = Some(arg["--project=".len()..].to_string());
+            // Note: You'd think this might work, but it's not actually supported.
+            // } else if arg == "--project" && i + 1 < args.len() {
+            //     project_arg = Some(args[i + 1].clone());
+        } else if arg == "--" {
+            break;
+        }
```

Comment:
This doesn't seem to handle my default way of launching Julia, which is `julia --proj`.

Reply:
Ta, I now handle `--proj`, `--proje`, `--projec`, and `--project`.